wallet/models.go

```diff
@@ -98,31 +99,37 @@ type GetAddressIndexRequest struct {
 // Index model
 type Index struct {
 	// Account index.
-	Major uint64 `json:"major"`
-	// Address index.
-	Minor uint64 `json:"minor"`
+	Major uint32 `json:"major"`
+	// Subaddress index.
+	Minor uint32 `json:"minor"`
 }
 
 // GetAddressIndexResponse represents the response model for GetAddressIndex
 type GetAddressIndexResponse struct {
-	// subaddress informations
+	// subaddress information
 	Index Index `json:"index"`
 }
 
 // CreateAddressRequest represents the request model for CreateAddress
 type CreateAddressRequest struct {
-	// Create a new address for this account.
-	AccountIndex uint64 `json:"account_index"`
```

Comment:
You can see the types actually used in this file here:
https://github.com/monero-project/monero/blob/master/src/wallet/wallet_rpc_server_commands_defs.h#L189